.github/workflows/_rocm-test.yml

```diff
@@ -96,6 +96,13 @@ jobs:
           github-token: ${{ secrets.GITHUB_TOKEN }}
           test-matrix: ${{ inputs.test-matrix }}
 
+      - name: Download pytest cache
```

Comment:
This is probably a no-op for ROCm as I don't think they have access to S3. As I don't see ROCm test is run inhttps://hud.pytorch.org/pr/100522, it's probably a good idea to do a rebase as it was restored from unstable since last week

Reply:
Rocm does seem to have access to S3 (at least the read command appears to work as per the "Download pytest cache" step in [this job](https://github.com/pytorch/pytorch/actions/runs/4921206763/jobs/8791141445)), though I'm surprised that there isn't a .pytest_cache folder found to upload despite there being a failure

Reply:
Rocm does seem to have access to S3 (at least the read command appears to work as per the "Download pytest cache" step in [this job](https://github.com/pytorch/pytorch/actions/runs/4921206763/jobs/8791141445)), though I'm surprised that there isn't a .pytest_cache folder found to upload despite there being a failure

Reply:
Oh man, so all jobs map the `$GITHUB_WORKSPACE` path to the `/var/lib/jenkins/workspace` folder. 

However, while in most linux jobs pytest uses that workspace folder as it's root folder, rocm uses `/var/lib/jenkins/pytorch` as it's root folder for some reason. That docker folder isn't mapped to any file on the drive and is thus inaccessible outside the docker environment!

(this can be fixed)

Reply:
Thanks for the context. For now I'm removing rocm support (we can look into adding it back later, since rocm machines do appear to have S3 access) and mac support (since they definitely  don't have S3 access).

For now let's focus on getting the basic Linux/Windows support